tests/acl_input/acl1.json

```diff
@@ -141,6 +141,57 @@
 					"config": {
 						"name": "everflowV6"
 					}
+				},
+				"DATAACL": {
+					"acl-entries": {
+						"acl-entry": {
+							"1": {
+								"config": {
+									"sequence-id": 1
+								},
+								"actions": {
+									"config": {
+										"forwarding-action": "DROP"
+									}
+								},
+								"ip": {
+									"config": {
+										"protocol": "IP_ICMP",
+										"source-ip-address": "::1/128",
+										"destination-ip-address": "::1/128"
+									}
+								},
+								"icmp": {
+									"config": {
+										"type": "4",
```

Comment:
>type [](start = 11, length = 4)

How did you test?

Reply:
The tests try to load various files, with `acl1.json` being a valid file. If the change is correct, this file should be successfully loaded and we should count 6 rules, as others have done previously.

Reply:
Agree you add check for `6`. Better to check more deeper details.

---
In reply to: [584383894](https://github.com/Azure/sonic-utilities/pull/1469#discussion_r584383894) [](ancestors = 584383894)